huaweicloud/services/dew/resource_huaweicloud_csms_secret.go

```diff
@@ -89,6 +89,24 @@ func ResourceCsmsSecret() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			"secret_type": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"enterprise_project_id": {
+				Type:     schema.TypeString,
+				Optional: true,
+				Computed: true,
+				ForceNew: true,
+			},
+			"event_subscriptions": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				MaxItems: 1,
```

Comment:
This field will be backfilled in the read method, please add `Computed` attribute

Reply:
the value can be left blank, not need computed